mzhadovich/task_05_CSS_background/background02.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<title>Домашнее задание - базовое #2</title>
+	<style type="text/css">
+		.rimnumber {
+			list-style-type: upper-roman;
+			width: 350px;
+		}
+		.rimnumber ul{
+			list-style-type: lower-roman;
+		}
+		.list {
+			width: 350px;
+		}
+		.cod {
+			width: 400px;
+			background-color: black;
+			color: green;
+			max-height: 4em;
```

Comment:
It wouldn't work here (hint: you need set absolute value for line-height and max-height)